datadog_checks_base/datadog_checks/base/utils/discovery/__init__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2019-present
```

Comment:
also : 

![image](https://user-images.githubusercontent.com/1266346/211316945-fcd8a2bf-47c2-41b3-b10c-b713493f7265.png)

😃

Reply:
Fortunately Alex is working on fixing that automatically!

Reply:
Could you update the year?

Reply:
Done